setup.py

```diff
@@ -32,7 +32,7 @@ def read_relative_file(filename):
         packages=['x509'],
         include_package_data=True,
         zip_safe=False,
-        install_requires=['setuptools>=1.1.6', 'flask', 'django-uuidfield',
+        install_requires=['setuptools>=1.1.6', 'flask', 'Django>=1.7',
```

Comment:
Actually I wonder if Django shouldn't be in `extra_require` instead. Any thought about this ?